notify/opsgenie/opsgenie.go

```diff
@@ -211,13 +219,44 @@ func (n *Notifier) createRequests(ctx context.Context, as ...*types.Alert) ([]*h
 			responders = append(responders, responder)
 		}
 
+		var visibleTos []opsGenieCreateMessageVisibleTo
+		for _, v := range n.conf.VisibleTo {
+			visibleTo := opsGenieCreateMessageVisibleTo{
+				ID:       tmpl(v.ID),
+				Name:     tmpl(v.Name),
+				Username: tmpl(v.Username),
+				Type:     tmpl(v.Type),
+			}
+
+			if visibleTo == (opsGenieCreateMessageVisibleTo{}) {
```

Comment:
Why would someone want to do this? Is it to catch badly written templates that produce no output?